include/rl_string_view.h

```diff
@@ -0,0 +1,9 @@
+#pragma once
+
+// Rely on the VW implementation of string_view
+#include "../ext_libs/vowpal_wabbit/vowpalwabbit/vw_string_view.h"
```

Comment:
A public header cant have a local repo dependent include path like this. This wont work when installed. 

It should be either `vowpalwabbit/vw_string_view.h` or `vw_string_view.h` with the appropriate include path made available via the VowpalWabbit::vw dependency

Reply:
we should change future_compat.h then too? how does it work with that?

Reply:
Yep, we need to fix that one too.

Reply:
OK I can fix both together in a separate PR too, how does that sound?

Reply:
Sounds good!